test/test_folder.py

```diff
@@ -36,6 +36,20 @@ def test_image_folder(self):
         outputs = sorted([dataset[i] for i in range(len(dataset))])
         self.assertEqual(imgs, outputs)
 
+        dataset = ImageFolder(Tester.root, loader=lambda x: x, is_valid_file=lambda x: True)
```

Comment:
can you make a slightly different `is_valid_file`? something like `'3' not in x`, so that it's not exactly the same test as above?

Reply:
`lambda x: x.lower().endswith("png")`  ok ?

Reply:
But this will return all images in the folder, right? The goal would be to precisely filter out one or two images to test this path

Reply:
Then I will modify the code below it as it checks for having all files in the dataset. ok?

Reply:
yes, thanks!

Reply:
done